### ISSUE TITLE
Create Online Demo

### ISSUE THREAD

**Issue by Gillespie59** [Gillespie59/eslint-plugin-angular#67]
Like http://eslint.org/demo/

**Comment by johnpapa** [comment]
demo should show all of the eslintrc options that cover the features you have implemented

**Comment by Gillespie59** [comment]
Yes. I would like the implement the possibility to enable/disable rules, and change the default configuration.
